src/components/molecules/FormFields/ServiceSelfDescription.tsx

```diff
@@ -40,13 +40,9 @@ export default function ServiceSelfDescription(
       props?.setStatus('loading')
 
       const parsedServiceSelfDescription = JSON.parse(rawServiceSelfDescription)
-      const signedServiceSelfDescription =
```

Comment:
@nklomp do we want to delete the support for old structure here? 
`{
sd: VerifiableCredential,
complianceCredential: VerifiableCredential
}`

Reply:
No